packages/phrases/src/locales/zh-cn.ts

```diff
@@ -181,71 +180,70 @@ const translation = {
-      enter_your_application_name: 'Enter your application name',
-      cancel: 'Cancel',
-      delete: 'Delete',
-      application_deleted: 'The application {{name}} deleted.',
-      save_success: 'Saved!',
+      save_changes: '保存修改',
+      more_options: '更多选项',
+      options_delete: '删除',
+      reminder: '提示',
+      delete_description: '此项操作不可撤销。他将会永久性地删除此应用。请输入 <span>{{name}}</span> 以确认.',
+      enter_your_application_name: '输入你的应用名称',
+      cancel: '取消',
+      delete: '删除',
+      application_deleted: '应用 {{name}} 成功删除.',
+      save_success: '已保存!',
       redirect_uri_required: 'You have to enter at least one redirect URI.',
     },
     api_resources: {
-      title: 'API Resources',
-      subtitle: 'Define APIs that you can consume from your authorized applications.',
-      create: 'Create API Resource',
-      api_name: 'API Name',
+      title: 'API 资源',
+      subtitle: '定义可以从已授权的应用程序中使用的 API.',
+      create: '创建 API 资源',
+      api_name: 'API 名词',
       api_identifier: 'API Identifier',
-      api_resource_created: 'The API resource {{name}} has been successfully created!',
+      api_resource_created: ' API 资源{{name}} 已被成功创建!',
     },
     api_resource_details: {
-      back_to_api_resources: 'Back to my API resources',
-      check_help_guide: 'Check Help Guide',
-      more_options: 'MORE OPTIONS',
-      options_delete: 'Delete',
-      settings: 'Settings',
+      back_to_api_resources: '返回我的 API 资源',
+      check_help_guide: '查看帮助引导',
+      more_options: '更多选项',
+      options_delete: '删除',
+      settings: '设置',
       save_changes: 'Save Changes',
       token_expiration_time_in_seconds: 'Token Expiration Time (in seconds)',
-      reminder: 'Reminder',
+      reminder: '提示',
       delete_description:
-        'This action cannot be undone. This will permanently delete the this API resource. Please enter the api resource name <span>{{name}}</span> to proceed.',
-      enter_your_api_resource_name: 'Enter your API resource name',
-      cancel: 'Cancel',
-      delete: 'Delete',
-      api_resource_deleted: 'The API Resource {{name}} deleted.',
-      save_success: 'Saved!',
+        '此项操作不可撤销。他将会永久性地删除此 API 资源。请输入 API 资源名称 <span>{{name}}</span>以确认.',
+      enter_your_api_resource_name: '输入 API 资源名称',
+      cancel: '取消',
+      delete: '删除',
+      api_resource_deleted: ' API 资源 {{name}} 已删除.',
+      save_success: '已保存!',
     },
     connectors: {
       title: '连接器',
-      subtitle: 'Setup connectors to enable passwordless and social sign in experience.',
+      subtitle: '设置连接器，开启无密码和社交登录。',
       create: '添加社会化登录',
       added: '已添加',
       set_up: '设置',
       tab_email_sms: '邮件/短信服务商',
       tab_social: '社会化登录',
       connector_name: '连接器',
       connector_type: '类型',
-      connector_status: '登录体验',
-      connector_status_in_use: '使用中',
-      connector_status_not_in_use: '未使用',
+      connector_status: '状态',
+      connector_status_enabled: '已启用',
+      connector_status_disabled: '已禁用',
```

Comment:
Should avoid changing keys. Or if we DO need to change, we change keys in both en and zh-cn files.